Modules/Test/classes/class.ilTestEvaluationUserData.php

```diff
@@ -183,9 +188,11 @@ public function getNumberOfQuestions(): int
         if (!is_object($this->passes[$questionpass])) {
             $questionpass = 0;
         }
```

Comment:
```suggestion
        if (!isset($this->passes[$questionpass])) {
            $questionpass = 0;
        }
```

Should this also be changed?